app/assets/stylesheets/custom.scss

```
@@ -82,3 +82,53 @@ input, textarea, select, .uneditable-input {
 input {
   height: auto !important;
 }
+
+/* Dropdown menu */
+
```

```suggestion
/* Dropdown menu */
```
Theo coding convention thì sau line comment ko có line trống mô gái